### PR TITLE
feat: add nav component

### PR DESCRIPTION
**Pull request by vermasavita**
- Deleted redundant code from the index.html
- Add navbar comp with search, action-icons, and logo

Link: https://deploy-preview-1--practical-dijkstra-12dc6e.netlify.app/

### CODE REVIEW

File: src/components/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+const Nav = () => {
+  return (
+    <div className ="container">
+      <header className="header">
+        <div className="logo">ByBook</div>
+
+        <div className="search-container">
+          <form action="">
+            <input id="search" type="search" placeholder="Search" />
+            <label for="search" className="bx bx-search-alt-2"></label>
```
Comment (AnkurChunekar) [inline src/components/Navbar/Navbar.jsx:11]: use `htmlFor` in place of `for` attribute, same as we use `className` for `class`

File: src/components/Navbar/Navbar.jsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
+import "./Navbar.css";
+const Nav = () => {
```
Comment (AnkurChunekar) [inline src/components/Navbar/Navbar.jsx:2]: It is a good practice to keep component name and file name similar, if you want you can change this to `const Navbar =` and then export Navbar.

**Review by AnkurChunekar — CHANGES_REQUESTED** [review]
Rest everything looks good.

File: src/components/Navbar/Navbar.jsx (hunk 2 above)
Comment (vermasavita) [reply src/components/Navbar/Navbar.jsx]: Done the change

File: src/components/Navbar/Navbar.jsx (hunk 1 above)
Comment (vermasavita) [reply src/components/Navbar/Navbar.jsx]: Changed

**Comment by vermasavita** [comment]
@AnkurChunekar Done the changes please approve it if it seems good to go

**Review by AnkurChunekar — APPROVED** [review]

**Review by RahullRawat — APPROVED** [review]

**Pull request merged**